packages/react/src/Pagination/Pagination.stories.tsx

```diff
@@ -1,38 +1,33 @@
-import React from 'react'
-import {Meta, StoryFn} from '@storybook/react'
+import type {Meta, StoryObj} from '@storybook/react'
 import {Pagination} from './Pagination'
 
-export default {
+const meta: Meta<typeof Pagination> = {
   title: 'Components/Pagination',
   component: Pagination,
   parameters: {
     layout: 'fullscreen',
   },
-} as Meta<typeof Pagination>
-
-const Template: StoryFn<typeof Pagination> = ({onPageChange, ...args}) => (
-  <main>
-    <Pagination
-      onPageChange={e => {
-        e.preventDefault()
-        const target = e.target as HTMLElement
-        // eslint-disable-next-line no-console
-        console.log('pressed: ', target.textContent)
-      }}
-      {...args}
-    />
-  </main>
-)
-
-export const Default = Template.bind({})
```

Comment:
The default story seems to be have been removed. Was that intended @joshfarrant?

Reply:
That's added back in 👍

Reply:
Actually, would you prefer that I align it with the example in our docs as it currently doesn't match

```tsx
export const Default: Story = {
  args: {
    pageCount: 10,
    currentPage: 5,
  },
}
```